test/helper.rb

```diff
@@ -167,6 +167,7 @@ def skip_unless(eng, bt: caller)
       when :fork    then MSG_FORK                       unless HAS_FORK
       when :unix    then MSG_UNIX                       unless Puma::HAS_UNIX_SOCKET
       when :aunix   then MSG_AUNIX                      unless Puma.abstract_unix_socket?
+      when :rack3   then "Skipped unless Rack >= 3.x"   unless ::Rack::RELEASE >= '3'
```

Comment:
Accidental?

Reply:
Should have been in the previous Rack 3 PR/commit.  I thought I might need it here, but there aren't any Rack objects...